services/collaboration/pkg/config/grpc.go

```diff
@@ -2,6 +2,8 @@ package config
 
 // GRPC defines the available grpc configuration.
 type GRPC struct {
-	Addr      string `yaml:"addr" env:"COLLABORATION_GRPC_ADDR" desc:"The bind address of the GRPC service." introductionVersion:"6.0.0"`
+	Addr     string `yaml:"addr" env:"COLLABORATION_GRPC_ADDR" desc:"The bind address of the GRPC service." introductionVersion:"6.0.0"`
+	Protocol string `yaml:"protocol" env:"COLLABORATION_GRPC_PROTOCOL" desc:"The transport protocol of the GRPC service." introductionVersion:"%%NEXT%%"`
```

Comment:
```suggestion
	Protocol string `yaml:"protocol" env:"COLLABORATION_GRPC_PROTOCOL" desc:"The transport protocol of the GRPC service. Supported values are 'tcp' and 'unix'." introductionVersion:"%%NEXT%%"`
```
I guess that protocol is not a collaboration thing but may affect more services. In case, consider using a global envvar additionally.

Reply:
no, some services must use tcp ... imo we can default to unix sockets for local / single node deployments. and the ocis chart can deal with the protocol if it really needs to cut out a service and run it independently.